<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>759</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icon/logo.png</normaloff>../Icon/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="PersonalInfo">
        <attribute name="title">
         <string>Personal Info</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QTableWidget" name="PersonalTable">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="AssociatedCompany">
        <attribute name="title">
         <string>Associated Company</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="AssociatedCompany_2">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>10</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="midLineWidth">
            <number>9</number>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::DashLine</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>CIN</string>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>9</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Company Name</string>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>9</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Begin Date</string>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>9</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>End Date</string>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>9</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ACTIVE Compliant</string>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>9</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(3,169,244);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;DIRECTOR PROFILE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="DINField">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>11</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>21</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Enter DIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Prefill">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255,193,7);
color: rgb(255, 255, 255);
border-radius:10</string>
          </property>
          <property name="text">
           <string>Prefill</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NameField">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>11</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Enter Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="FatherName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>11</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="placeholderText">
           <string>Father's Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DOB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>11</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="placeholderText">
           <string>Date of Birth</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
